modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/rel/AbstractExecutionTest.java

```diff
@@ -100,14 +100,18 @@ protected ExecutionContext<T> executionContext(boolean withDelays) {
                     false,
                     0);
 
-            StripedThreadPoolExecutor stripedThreadPoolExecutor = (StripedThreadPoolExecutor) IgniteTestUtils.getFieldValue(
+            StripedThreadPoolExecutor stripedThreadPoolExecutor = IgniteTestUtils.getFieldValue(
```

Comment:
Why don't just pass executor to execution context in "@BeforeTest" method?
Why does thread poll's concurrency level differs in `QueryTaskExecutorImpl` and in `IgniteTestStripedThreadPoolExecutor` ?